packages/network-controller/tests/fake-provider.ts

```diff
@@ -1,4 +1,4 @@
-import { SafeEventEmitterProvider } from '@metamask/eth-json-rpc-provider/dist/safe-event-emitter-provider';
+import { SafeEventEmitterProvider } from '@metamask/eth-json-rpc-provider';
```

Comment:
See line 85:
```ts
export class FakeProvider extends SafeEventEmitterProvider {
```

Importing `SafeEventEmitterProvider` as a type breaks that line, because it's being used as a class.

This is why the previous import statement pointed directly to `node_modules/.../dist/...` instead of using the `SafeEventEmitterProvider` type exported by the package. 

That import statement can no longer be used (it won't point to `node_modules` with `eth-json-rpc-provider` in `packages/`) which is why the export from `eth-json-rpc-provider` needs to be fixed from type to class.

Reply:
Ah, I see. Thank you for explaining, somehow that was not connecting.